### PR TITLE
improve CONTRIBUTING.md about Pull Request

### PR DESCRIPTION
**Pull request by zxcpoiu**
**motivation**

Lots of people does not sure what to do after a PR.
common questions like:

* should I keep rebasing this PR to the latest master branch ?
* should I keep squashing all commits in a PR into single one?

This may clear it a bit and makes people less nervous :scream: 

**Test plan (required)**

### CODE REVIEW

**Comment by facebook-github-bot-5** [comment]
By analyzing the blame information on this pull request, we identified @geirman and @mxstbr to be potential reviewers.

**Comment by zxcpoiu** [comment]
thanks @satya164

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -20,16 +20,21 @@ The core team will be monitoring for pull requests. When we get one, we'll run s
...
 3. If you've changed APIs, update the documentation.
 4. If you've updated the docs, verify the website locally and submit screenshots if applicable
-```
-$ cd website
-$ npm install && npm start
-go to: http://localhost:8079/react-native/index.html
-```
+
+    ```
+    $ cd websit
```
Comment (ide) [inline CONTRIBUTING.md]: website

**Comment by zxcpoiu** [comment]
good catch @ide thanks

**Comment by satya164** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-4** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/1039017896183392/int_phab) to review.

**Pull request closed without merging**